### PR TITLE
imporvements for loadings aysrepos actors templaterepos

### PR DESCRIPTION
**Pull request by xmonader**

### CODE REVIEW

**Comment by zaibon** [comment]
I don't specially like the fact that we have to use `find`.  
 Also I would like to see how https://github.com/Jumpscale/jumpscale_core8/issues/842 will impact on the overall perfs

**Comment by rkhamis** [comment]
@zaibon already the `find` cut down the scanning time to less than a third.

I'll merge for optimizati

File: lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py [hunk 1]
```diff
@@ -1,65 +1,22 @@
 from JumpScale import j
 from JumpScale.baselib.atyourservice81.lib.ActorTemplate import ActorTemplate
 import asyncio
+import os
```
Comment (rkhamis) [inline lib/JumpScale/baselib/atyourservice81/lib/TemplateRepo.py]: Not sure we need this import

**Comment by zaibon** [comment]
but isn't this commit will conflict with #842 ?
From what I see #842 will remove this code altogether

File: lib/JumpScale/baselib/atyourservice81/lib/AtYourServiceRepo.py [hunk 2]
```diff
@@ -70,22 +70,14 @@ def _searchAysRepos(self, path):
...
-            if base in [".git", ".hg", ".github"]:
-                return False
-            depth = len(j.sal.fs.pathRemoveDirPart(path, arg[0]).split("/"))
-            # print("%s:%s" % (depth, j.sal.fs.pathRemoveDirPart(path, arg[0])))
-            if depth < 6:
-                return True
-            return False
+        cmd = """find %s \( -wholename '*.ays' \) -exec readlink -f {} \;""" % (path)
+        rc, out, err = j.sal.process.execute(cmd, die=False, showout=False)
+        for reponame in out.splitlines():
```
Comment (rkhamis) [inline lib/JumpScale/baselib/atyourservice81/lib/AtYourServiceRepo.py]: Maybe we should add a strip and a check that the line is not empty here as well

**Pull request merged**